.github/workflows/ci.yml

```diff
@@ -321,12 +321,23 @@ jobs:
           MAPDL_IMAGE: ${{ env.DOCKER_PACKAGE }}:${{ matrix.mapdl-version }}
 
       - name: "Pull, launch, and validate Ubuntu MAPDL service from private"
-        run: .ci/start_mapdl_ubuntu.sh
+        run: |
+          touch tmpfile.sh
+          head -n 3 .ci/start_mapdl.sh > tmpfile.sh
+          echo -e "    --entrypoint '/bin/bash' \\" >> tmpfile.sh
+          head -n 14 .ci/start_mapdl.sh | tail -n 11 >> tmpfile.sh
+          echo -e "    -w /jobs \\" >> tmpfile.sh
+          echo -e "    -u=0:0 \\" >> tmpfile.sh
+          echo -e "    $MAPDL_IMAGE /ansys_inc/v222/ansys/bin/mapdl -grpc -dir /jobs -smp -np 2 > log.txt & \\" >> tmpfile.sh
+          tail -n 2 .ci/start_mapdl.sh >> tmpfile.sh
+          bash tmpfile.sh
+          rm tmpfile.sh
```

Comment:
Interesting approach... but if this is going to be the solution, we might stick to the fact that we need a different bash script. I'll look at it more in detail tomorrow.